carta/session.py

```diff
@@ -368,11 +368,13 @@ def open_image(self, path, hdu=""):
             The path to the image file, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
         hdu : {1}
             The HDU to select inside the file.
+        complex : {2}
+            Complex value to open a comlex-valued image. The valid options are ``AMPLITUDE``, ``PHASE``, ``REAL`` and ``IMAGINARY``. Default is ``AMPLITUDE``.
```

Comment:
Update this description as well (but you won't need to update the type information, because the validator fills that in -- maybe check if what I suggested in `new` matches the validator's description).

Reply:
Done in 0b989dc.